src/server/controllers/verification/verification.common.ts

```diff
@@ -17,13 +17,13 @@ import {
   useAllSources,
 } from "@ethereum-sourcify/lib-sourcify";
 import { Session } from "express-session";
-import Web3 from "web3";
 import { AbiConstructor, AbiParameter } from "abitype";
 import QueryString from "qs";
 import fetch from "node-fetch";
 import { IVerificationService } from "../../services/VerificationService";
 import { IRepositoryService } from "../../services/RepositoryService";
 import { ContractMeta, ContractWrapper } from "../../common";
+import { id as keccak256str } from "ethers";
```

Comment:
I see sometimes you are using id and other times keccak256, they are the same thing as far as I know (see the example here: https://docs.soliditylang.org/en/latest/path-resolution.html#id12) it would be more consistent to always use the same. But it's not bad

Reply:
Nope they are different. 

Ethersjs has 
1) `keccak256` that takes a `BytesLike` https://docs.ethers.org/v6/api/crypto/#keccak256
2) `id` that takes a string (UTF-8) https://docs.ethers.org/v6/api/hashing/#id

Reply:
Is the link you provided correct? Looks out of the context

Reply:
sorry, wrong link because I thought it copied it :D the real link was from the ethers docs (but actually version 5). in the example it was stated that they are the same thing after converting the string into bytes. https://docs.ethers.org/v5/api/utils/hashing/#utils-id


```
// If needed, convert strings to bytes first:
utils.keccak256(utils.toUtf8Bytes("hello world"))
// '0x47173285a8d7341e5e972fc677286384f802f8ef42a5ec5f03bbfa254cb01fad'

// Or equivalently use the identity function:
utils.id("hello world")
// '0x47173285a8d7341e5e972fc677286384f802f8ef42a5ec5f03bbfa254cb01fad'
```

Reply:
Ok so we are talking about the same thing? What's your concern here? Since `id` itself is not really descriptive I'm using `keccak256str`. It is different than `keccak256`

Reply:
Easy, I was just confused of having two different keccak functions. It is good like this